can not run the hello-world example in winodws

I followed the docs and do something like below:
npm install -g yo
npm install -g generator-react-server
yo react-server
npm run start

All the things I expected is it start server at port 3000
But something wrong.
I get some error like this(BTW,my OS is windows 10):

> 
"C:\Program Files (x86)\JetBrains\WebStorm 11.0.3\bin\runnerw.exe" "C:\Program Files\nodejs\node.exe" "C:\Program Files\nodejs\node_modules\npm\bin\npm-cli.js" run-script start

> reactserver@0.0.1 start C:\mydev\reactserver
> react-server start

>2016-08-22T03:14:36.445Z - warning: [react-server-cli.src.logProductionWarnings] PRODUCTION WARNING: the following current settings are discouraged in production environments. (If you are developing, carry on!):
2016-08-22T03:14:36.449Z - warning: [react-server-cli.src.logProductionWarnings] -- Minification is disabled. To enable, set minify to true (--minify at the command-line) or set NODE_ENV=production.
2016-08-22T03:14:36.450Z - warning: [react-server-cli.src.logProductionWarnings] -- Long-term caching is disabled. To enable, set longTermCaching to true (--long-term-caching at the command-line) or set NODE_ENV=production to turn on.
2016-08-22T03:14:36.450Z - info: [react-server-cli.src.logProductionWarnings] NODE_ENV is set to undefined
2016-08-22T03:14:36.531Z - notice: [react-server-cli.src.commands.start] Starting servers...
There were errors in the JavaScript compilation.
2016-08-22T03:14:40.047Z - info: [react-server-cli.src.commands.start] Starting HTML server...
./__clientTemp/routes_client.js
eactserver
eact-server-core-middlewareindex.js' in C:\mydev\reactserver\__clientTemp
eactserver
eact-server-core-middlewareindex.js in C:\mydev\reactserver\__clientTemp
  looking for modules in C:\mydev\reactserver\node_modules
eactserver
eact-server-core-middlewareindex.js doesn't exist (module as directory)
eactserver
eact-server-core-middlewareindex.js in C:\mydev\reactserver\node_modules
      resolve file
eactserver
eact-server-core-middlewareindex.js doesn't exist
eactserver
eact-server-core-middlewareindex.js.webpack.js doesn't exist
eactserver
eact-server-core-middlewareindex.js.web.js doesn't exist
eactserver
eact-server-core-middlewareindex.js.js doesn't exist
eactserver
eact-server-core-middlewareindex.js.json doesn't exist
eactserver
eact-server-core-middlewareindex.js]
eactserver
eact-server-core-middlewareindex.js]
eactserver
eact-server-core-middlewareindex.js.webpack.js]
eactserver
eact-server-core-middlewareindex.js.web.js]
eactserver
eact-server-core-middlewareindex.js.js]
eactserver
eact-server-core-middlewareindex.js.json]
 @ ./__clientTemp/routes_client.js 7:23-105 8:24-106
./__clientTemp/routes_client.js
Module not found: Error: Cannot resolve module '..pageshello-world' in C:\mydev\reactserver\__clientTemp
resolve module ..pageshello-world in C:\mydev\reactserver\__clientTemp
  looking for modules in C:\mydev\reactserver\node_modules
    C:\mydev\reactserver\node_modules\..pageshello-world doesn't exist (module as directory)
    resolve 'file' ..pageshello-world in C:\mydev\reactserver\node_modules
      resolve file
        C:\mydev\reactserver\node_modules\..pageshello-world doesn't exist
        C:\mydev\reactserver\node_modules\..pageshello-world.webpack.js doesn't exist
        C:\mydev\reactserver\node_modules\..pageshello-world.web.js doesn't exist
        C:\mydev\reactserver\node_modules\..pageshello-world.js doesn't exist
        C:\mydev\reactserver\node_modules\..pageshello-world.json doesn't exist
[C:\mydev\reactserver\node_modules\..pageshello-world]
[C:\mydev\reactserver\node_modules\..pageshello-world]
[C:\mydev\reactserver\node_modules\..pageshello-world.webpack.js]
[C:\mydev\reactserver\node_modules\..pageshello-world.web.js]
[C:\mydev\reactserver\node_modules\..pageshello-world.js]
[C:\mydev\reactserver\node_modules\..pageshello-world.json]
 @ ./__clientTemp/routes_client.js 19:7-21:9 20:27-58
2016-08-22T03:14:40.148Z - error: [react-server-cli.src.commands.start]  message=There were errors in the JavaScript compilation., stack=Error: There were errors in the JavaScript compilation.
    at handleCompilationErrors (C:\mydev\reactserver\node_modules\react-server-cli\target\handleCompilationErrors.js:25:10)
    at C:\mydev\reactserver\node_modules\react-server-cli\target\commands\start.js:180:55
    at Compiler.<anonymous> (C:\mydev\reactserver\node_modules\webpack\lib\Compiler.js:194:14)
    at Compiler.emitRecords (C:\mydev\reactserver\node_modules\webpack\lib\Compiler.js:282:37)
    at Compiler.<anonymous> (C:\mydev\reactserver\node_modules\webpack\lib\Compiler.js:187:11)
    at C:\mydev\reactserver\node_modules\webpack\lib\Compiler.js:275:11
    at Compiler.applyPluginsAsync (C:\mydev\reactserver\node_modules\tapable\lib\Tapable.js:60:69)
    at Compiler.afterEmit (C:\mydev\reactserver\node_modules\webpack\lib\Compiler.js:272:8)
    at Compiler.<anonymous> (C:\mydev\reactserver\node_modules\webpack\lib\Compiler.js:267:14)
    at C:\mydev\reactserver\node_modules\webpack\node_modules\async\lib\async.js:52:16

Process finished with exit code 0

--- a/packages/generator-react-server/generators/app/templates/routes.js
+++ b/packages/generator-react-server/generators/app/templates/routes.js
@@ -1,10 +1,11 @@
+const path = require('path');
 
 module.exports = {
 	routes: {
 		HelloWorld: {
 			path: ['/'],
 			method: 'get',
-			page: './pages/hello-world',
+			page: path.join('.', 'pages', 'hello-world'),
 		},
 	},
 };

